fili-core/src/main/java/com/yahoo/bard/webservice/data/config/names/DimensionSearchLogicalTableName.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.config.names;
+
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+/**
+ * Hold the list of logical table names.
+ */
+public enum DimensionSearchLogicalTableName implements TableName {
```

Comment:
What are we using this for?

Reply:
We're using this to construct a table for the DimensionSearchServlet tests: https://github.com/ldkge/fili/commit/fa7c967a82e3b7b93bbfd149da1009fe96741fcc#diff-7010143af714e22d45c8c0746d2c3b5eR30